examples/ag-ui/src/App.tsx

```
@@ -0,0 +1,311 @@
+import { useEffect, useMemo, useState } from 'react'
+import { fetchServerSentEvents } from '@tanstack/ai-react'
+import type { UIMessage } from '@tanstack/ai-react'
+import {
+  Chat,
+  ChatInput,
+  ChatMessage,
+  ChatMessages,
+} from '@tanstack/ai-react-ui'
```

ESLint

[error] `@tanstack/ai-react-ui` import should occur before type import of `@tanstack/ai-react`

(import/order)